form.js

```diff
@@ -0,0 +1,18 @@
+// VALIDATIONA FORM
+
+const error = document.getElementById('error-message');
+const form = document.querySelector('.contact-form');
+const userEmail = document.getElementById('user-email');
+const errorMessage = 'Please write your email address in lower case';
+// CODE
+form.addEventListener('submit', (event) => {
+  // preventing from submittion
+  event.preventDefault();
+  const userInput = userEmail.value;
+  if (userInput !== userInput.toLowerCase()) {
+    error.innerText = errorMessage;
+  } else {
+    error.innerText = '';
+    form.submit();
```

Comment:
- [ ] You are having an error here whenever the form is good from validations but it cannot actually submit, kindly check it 
![image](https://github.com/Tarto-4/Tarto-4.github.io/assets/62705947/e037a419-840b-4082-be7e-e848f3c43d1a)